### PR TITLE
fix warning from license and npm

### PR DESCRIPTION
**Pull request by FeelyChau**
Fix the warn as follows:
```sh
npm WARN lifecycle The node binary used for scripts is /var/folders/tq/cmmgdq_143jbd5w_drrdl8km0000gp/T/yarn--1618122759637-0.3013471152358005/node but npm is using /path/to/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.
```
And license string.

### CODE REVIEW

**Review by WenheLI — APPROVED** [review]
LGTM

File: packages/cli/package.json [hunk 1]
```diff
@@ -5,18 +5,18 @@
   "main": "dist/bin/pipcook.js",
   "scripts": {
     "test": "ava",
-    "build": "npm run clean && npm run compile",
+    "build": "yarn clean && yarn compile",
```
Comment (yorkie) [inline packages/cli/package.json:8]: yarn should not be required.

Comment (FeelyChau) [reply packages/cli/package.json:8]: Fixed.

**Review by WenheLI — DISMISSED** [review]

**Review by rickycao-qy — APPROVED** [review]
LGTM

**Review by yorkie — APPROVED** [review]

**Pull request merged**